flow/record/base.py

```diff
@@ -578,19 +566,19 @@ def extend(self, fields: Sequence[Tuple[str, str]]) -> RecordDescriptor:
         new_fields = list(self.get_field_tuples()) + fields
         return RecordDescriptor(self.name, new_fields)
 
-    def get_field_tuples(self) -> Tuple[Tuple[str, str]]:
+    def get_field_tuples(self) -> tuple[tuple[str, str]]:
         """Returns a tuple containing the (typename, name) tuples, eg:
 
         (('boolean', 'foo'), ('string', 'bar'))
 
         Returns:
-            Tuple of (typename, name) tuples
+            tuple of (typename, name) tuples
```

Comment:
```suggestion
            Tuple of (typename, name) tuples
```
Bit too eager find & replace.